File: intro/php/fizzBuzz.php
<?php

declare(strict_types=1);


function fizzBuzz(int $n): array {
    $result = [];
    for ($i = 1; $i <= $n; $i++) {
        if ($i % 15 === 0) {
            $result[] = 'fizzbuzz';
        } elseif ($i % 3 === 0) {
            $result[] = 'fizz';
        } elseif ($i % 5 === 0) {
            $result[] = 'buzz';
        } else {
            $result[] = $i;
        }
    }
    return $result;
}

File: Intro/go/fizzbuzz.go
package main

import (
	"fmt"
)

func fizzBuzz(n int) []any {
	result := make([]any, 0, n)
	for i := 1; i <= n; i++ {
		switch {
		case i%15 == 0:
			result = append(result, "fizzbuzz")
		case i%3 == 0:
			result = append(result, "fizz")
		case i%5 == 0:
			result = append(result, "buzz")
		default:
			result = append(result, i)
		}
	}
	return result
}

func main() {
	fmt.Println(fizzBuzz(16))
}